articles/active-directory/develop/msal-net-web-browsers.md

```diff
@@ -98,14 +98,11 @@ IPublicClientApplication pca = PublicClientApplicationBuilder
 
 ### Linux and MAC
 
-On Linux, MSAL.NET will open the default OS browser using the xdg-open tool. To troubleshoot, run the tool from a terminal, for example, `xdg-open "https://www.bing.com"`. On Mac, the browser is opened by invoking `open <url>`.
+On Linux, MSAL.NET will open the default OS browser using the a tool such as `xdg-open`. Opening the browser as `sudo` does is not supported, and MSAL will throw an exception. On Mac, the browser is opened by invoking `open <url>`.
```

Comment:
```suggestion
On Linux, MSAL.NET opens the default OS browser with a tool like [xdg-open](http://manpages.ubuntu.com/manpages/focal/man1/xdg-open.1.html). Opening the browser with root privileges as `sudo` does is unsupported by MSAL and will cause MSAL to throw an exception.

On macOS, the browser is opened by invoking `open <url>`.
```

Reply:
Not sure about "root privileged like `sudo`". There is only `sudo` on Linux.

Reply:
OK, still need some clarity on your original statement that mentions `sudo`.

When executed without `-u`, the _usual_ default behavior of `sudo` to execute the command as root, and thus, with root privileges.

From the `sudo` man page:

```console
...
DESCRIPTION
     sudo allows a permitted user to execute a command as the superuser or another user, as specified by the security policy.  The invoking user's real (not
     effective) user-ID is used to determine the user name with which to query the security policy.
...
      -u user, --user=user
                 Run the command as a user other than the default target user (usually root).
```

This in particular is the confusing part of your original change:

> Opening the browser as `sudo` does is not supported

With the "does" in there, it sounds like you're saying that's _all_ that `sudo` does, ever, which is incorrect. I was trying to clarify what behavior of `sudo` in this situation was the most important, so if you have another way to do so, I'm all ears. :slightly_smiling_face:

Reply:
@bgavrilMS How do you want to handle this one? Since opening the browser isn't _only_ thing `sudo` does, how about this:

> On Linux, MSAL.NET opens the default OS browser with a tool like [xdg-open](http://manpages.ubuntu.com/manpages/focal/man1/xdg-open.1.html). Opening the browser with `sudo` is unsupported by MSAL and will cause MSAL to throw an exception.

I've updated the suggestion in https://github.com/MicrosoftDocs/azure-docs/pull/88259/commits/5264bbdbb570c29466a7cd9d1e9317e65da89a4f#r813053456 to reflect the newly suggested text for that line.

Reply:
That sounds good, thanks @mmacy for the update, this fell off my radar.